src/vehicle_gui.cpp

```diff
@@ -1651,14 +1663,17 @@ struct VehicleListWindow : public BaseVehicleListWindow {
 		/* Set up the window widgets */
 		this->GetWidget<NWidgetCore>(WID_VL_LIST)->tool_tip = STR_VEHICLE_LIST_TRAIN_LIST_TOOLTIP + this->vli.vtype;
 
+		NWidgetStacked* nwi = this->GetWidget<NWidgetStacked>(WID_VL_CAPTION_SELECTION);
```

Comment:
Minor nitpick in case we need to rebase: `NWidgetStacked *nwi`

Reply:
Fixed.